Dockerfile

```diff
@@ -1,9 +1,18 @@
+
+# this Dockerfile is expected to run with a checkout
+# of the vzlogger source as the build context (working directory),
+# which it will copy into the container and then build.
+
 ############################
 # STEP 1 build executable binary
 ############################
 
 FROM alpine:latest as builder
 
+COPY . /vzlogger
```

Comment:
pulling this up here breaks docker-cache caching of the package installations below :(

maybe just copy CMakeLists.txt to a dummy location, it changes rarely so the impact on the cache should be neglegible.
or accept to fail less early.